src/index.tsx

```diff
@@ -30,20 +26,22 @@ if (process.env.NODE_ENV === "development") {
   });
 }
 
-function init(firebaseConfig) {
-  firebase.initializeApp(firebaseConfig);
+function init(firebaseConfig:FirebaseOptions) {
+  const app = initializeApp(firebaseConfig)
```

Comment:
One thing to note. Originally you created your own typing for firebaseConfig. Creating your own typing should be your last best. Firebase is written in TS, so it should have all its own typings. So in this case, you can just import the typing directly for firebase as I did here